frontend/src/components/auth/Login.tsx

```diff
@@ -98,29 +136,29 @@ const Login = () => {
           >
             Log In
           </button>
-          <ResetPassword email={email} />
-
-          <GoogleLogin
-            clientId="175399577852-6hll8ih9q1ljij8f50f9pf9t2va6u3a7.apps.googleusercontent.com"
-            buttonText="Google"
-            onSuccess={(response: GoogleResponse): void => {
-              if ("tokenId" in response) {
-                onGoogleLoginSuccess(response.tokenId);
-              } else {
-                alert(JSON.stringify(response));
-              }
-            }}
-            onFailure={onFailure}
-          />
-        </div>
-        <div>
           <button
             className="btn btn-primary"
             type="button"
             onClick={onSignUpClick}
           >
             Sign Up
           </button>
+          {!isSignup && <ResetPassword email={email} />}
+
+          {!isSignup && (
```

Comment:
We can always display the google login - it can handle both sign up and login in one button :)